stack-8.6.5.yaml

```diff
@@ -5,14 +5,14 @@ compiler-check: match-exact
 packages:
 - .
 
-# Force use of newer HsOpenSSL, used by http-iostreams
-drop-packages:
-- HsOpenSSL
+# # Force use of newer HsOpenSSL, used by http-io-streams
+# drop-packages:
+# - HsOpenSSL
 
 extra-deps:
   # HSOpenSSL-0.11.6.1: Windows: Use libcrypto instead of eay; allow pkg-config.
 - HsOpenSSL-0.11.6.1
```

Comment:
Weird, why does in Windows CI `HsOpenSSL-0.11.7.2` get picked? https://github.com/hackage-trustees/hackage-cli/runs/6712301152?check_suite_focus=true#step:9:993